src/pages/taskHistory/BuildVariantSelector.tsx

```diff
@@ -48,35 +45,25 @@ const BuildVariantSelector: React.VFC<BuildVariantSelectorProps> = ({
 
   const { buildVariantsForTaskName } = data || {};
 
-  const handleSearch = (options: BuildVariantTuple[], match: string) =>
-    options.filter(
-      (option) =>
-        option.buildVariant.includes(match) ||
-        option.displayName.includes(match)
-    );
-
   return (
     <Container>
-      <SearchableDropdown
+      <Combobox
         data-cy="build-variant-selector"
         label="Build Variant"
-        valuePlaceholder="Select build variant to view"
+        placeholder="Select build variant to view"
```

Comment:
I know this was here before but "to view" feels redundant to me
```suggestion
        placeholder="Select build variants"
```